_layouts/tour.html

```diff
@@ -28,7 +28,9 @@ <h1>{{page.title}}</h1>
         </div>
     </div>
     <div class="row pt-5">
-        <h1>2023春节游园会活动介绍</h1>
+        <div class="underline-col">
```

Comment:
https://getbootstrap.com/docs/4.0/utilities/text/
```suggestion
        <div class="text-center">
```

Also, what if you combine `text-center` into `<div class="row pt-5">` (i.e., `<div class="row pt-5 text-center">`)?

Reply:
Updated.